obspy/signal/spectral_estimation.py

```diff
@@ -2184,23 +2207,27 @@ def __plot_coverage(self, ax):
         ax.autoscale_view()
 
 
-def get_nlnm():
+def get_nlnm(mod_fl):
     """
     Returns periods and psd values for the New Low Noise Model.
     For information on New High/Low Noise Model see [Peterson1993]_.
     """
-    data = np.load(NOISE_MODEL_FILE)
+    # The 'mod_fl' argument chooses between sesimic or infrasound noise models
```

Comment:
This should be inside of the docstring, not as a separate comment

Reply:
I think this is resolved after a created new definitions to read the infrasound models per megies's advice.